integrationTests/realcomponents/processorRunner.go

```diff
@@ -453,6 +469,7 @@ func (pr *ProcessorRunner) createProcessComponents(tb testing.TB) {
 		StatusCoreComponents:    pr.StatusCoreComponents,
 		TxExecutionOrderHandler: txExecutionOrderHandler,
 		RunTypeComponents:       pr.RunTypeComponents,
+		EnableEpochsFactory:     pr.RunTypeCoreComponents.EnableEpochsFactoryCreator(),
```

Comment:
Same remark as above

Reply:
RunTypeCoreComponents were made for CoreComponents, not really for ProcessComponents, that's why I just pass the factory to process components. Also, it's only one component needed from runType core.